meta-lmp-base/recipes-containers/wayland-cdi/wayland-cdi_0.1.bb

```diff
@@ -0,0 +1,24 @@
+SUMMARY = "Sets up a CDI device entry for the Wayland display"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+inherit allarch systemd
+
+RDEPENDS:${PN} = "systemd"
+
+SRC_URI = "file://wayland-cdi.service \
+    file://wayland-cdi-generate \
+"
+
+PACKAGE_ARCH = "${MACHINE_ARCH}"
```

Comment:
Also fine to drop this one, not really machine specific (for what it currently does).

Reply:
fixed in force commit. also rebased on latest main.